tensordict/prototype/tensordictclass.py

```diff
@@ -180,11 +176,12 @@ def wrapped_func(*args, **kwargs):
                 return wrapped_func
 
         def __getitem__(self, item):
+            if isinstance(item, str) and item in self.tensordict.keys():
```

Comment:
what if it is a string but not in the tensordict?
Also a tuple of strings should be prohibited too